src/api/app/models/workflow/step/branch_package_step.rb

```diff
@@ -16,6 +16,16 @@ def target_project_base_name
     step_instructions[:target_project]
   end
 
+  def target_project
+    Project.find_by(name: target_project_name)
+  end
+
+  def add_repositories?
+    return true if step_instructions[:add_repositories].blank?
```

Comment:
```suggestion
    return false if step_instructions[:add_repositories].blank?
```

Reply:
@hennevogel I would find it more intuitive to return false if there is nothing to add

Reply:
ah I see, sorry now I get it

Reply:
```suggestion
    step_instructions[:add_repositories].blank? || step_instructions[:add_repositories] == 'enable'
```
I find this option more intuitive.